### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by hrittikhere**
This PR updates readme.md!

### CODE REVIEW

**Comment by hrittikhere** [comment]
@Ompragash please review!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,54 @@
 # Continuous Delivery Pipeline Sample
 
-## Follow this for easy navigation!
-***
+![Cover Image CD Pipeline](Images/harness_cover_image.png)
+
+This sample repository shows how to use the [Harness Continuous Delivery Pipeline](https://harness.io/products/continuous-delivery) to deploy a nginx image to Google Kubernetes Engine using YAML based pipeline creation.
```
Comment (Ompragash) [inline README.md:5]:
```suggestion
This sample repository shows how to use the [Harness Continuous Delivery Pipeline](https://harness.io/products/continuous-delivery) to deploy an Nginx image to Google Kubernetes Engine using YAML based pipeline creation.
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+
+
+
+## Getting Started
+
+Use this README to get started with our sample pipeline repository for CD. This guide outlines the basics of getting started with the Harness CD and provides a full YAML to set up your pipeline. This sample doesn’t include configuration options for in-depth steps and configuring the pipeline for other services, but only focuses on Kubernetes.
+
+
+Setting up and running the pipeline will take about 30 minutes, and by the end, you can run an nginx image on your cluster with Harness!
```
Comment (Ompragash) [inline README.md:14]:
```suggestion
Setting up and running the pipeline will take about 30 minutes, and by the end, you can run an Nginx image on your cluster with Harness!
```

**Review by Ompragash — CHANGES_REQUESTED** [review]
Thanks for the PR @hrittikhere! Added minor changes and rest looks perfect!

**Review by Ompragash — APPROVED** [review]

**Pull request merged**